### DISCUSSION TITLE
Help with translation

### DISCUSSION THREAD

**Discussion by ezellohar** [windingwind/zotero-actions-tags#179] [Q&A] [answered]
Line 20 in preferences locale file has: "toggle tags". what does it actually do? I'm having troubles finding a suitable translation, thanks

**Comment by windingwind** [comment, marked as answer]
add a tag if it does not exist on the item and otherwise, remove the tag.

**Reply by ezellohar** [reply]
Thanks, that's hard to translate since we don't have a specific word with that meaning lol, I'm guess I'll just add both words
